pkg/nmi/server/server.go

```diff
@@ -490,3 +460,13 @@ func validateResourceParamExists(resource string) bool {
 	}
 	return true
 }
+
+// Create the client config. Use kubeconfig if given, otherwise assume in-cluster.
+func buildConfig() (*rest.Config, error) {
+	kubeconfigPath := os.Getenv("KUBECONFIG")
```

Comment:
>KUBECONFIG [](start = 30, length = 10)

Why not add optional flag instead?

Reply:
Could you explain a little more for this comment? Thanks.

---
In reply to: [383475214](https://github.com/bowu24/aad-pod-identity/pull/1#discussion_r383475214) [](ancestors = 383475214)

Reply:
you can define "kubeconfig" flag in nmi main()

Look at our deployment agent for example

---
In reply to: [386669543](https://github.com/bowu24/aad-pod-identity/pull/1#discussion_r386669543) [](ancestors = 386669543,383475214)

Reply:
Got it. Updated.

---
In reply to: [386746627](https://github.com/bowu24/aad-pod-identity/pull/1#discussion_r386746627) [](ancestors = 386746627,386669543,383475214)